server/lib/realtime/application.ex

```diff
@@ -35,6 +35,20 @@ defmodule Realtime.Application do
 
     replication_mode = Application.fetch_env!(:realtime, :replication_mode)
 
+    # When UNSAFE_ALL_DEBUG_LOGS is set, we show log-level of debug.
+    # This is marked unsafe, because it should not be used in production,
+    # and is not generally guaranteed that DEBUG-level logs will be performant.
+    unsafe_all_debug_logs =
+      System.get_env(
+        "UNSAFE_ALL_DEBUG_LOGS",
```

Comment:
```suggestion
        "UNSAFE_SHOW_DEBUG_LOGS",
```

Just a suggestion. I needed to read your comment to understand _what_ this does. Hopefully adding a verb makes it a bit clearer

Reply:
very fair, will update.

Reply:
just stumbled across this..
maybe keep `unsafe_all_debug_logs` and `UNSAFE_SHOW_DEBUG_LOGS` also in sync :)